src/Cache/CacheKeyGenerator.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace ScriptFUSION\Porter\Cache;
+
+interface CacheKeyGenerator
+{
+    /**
+     * @param string $source
+     * @param array $optionsSorted Key sorted options.
+     *
+     * @return string A PSR-6 compatible cache key.
+     */
+    public function generateCacheKey($source, array $optionsSorted);
```

Comment:
I think `$sortedOptions` reads more naturally, don't you? Applies to `JsonCacheKeyGenerator `, too.